front/app/modules/commercial/content_builder/admin/components/ContentBuilderToolbox/index.tsx

```diff
@@ -3,31 +3,99 @@ import React from 'react';
 // craft
 import { useEditor, Element } from '@craftjs/core';
 import Container from '../CraftComponents/Container';
+import Text from '../CraftComponents/Text';
 
 // intl
 import { injectIntl } from 'utils/cl-intl';
 import { InjectedIntlProps } from 'react-intl';
 import messages from '../../messages';
-import { Box } from '@citizenlab/cl2-component-library';
+import { Box, Icon } from '@citizenlab/cl2-component-library';
+
+// Styling
+import styled from 'styled-components';
+import { colors } from 'utils/styleUtils';
+
+const ToolboxSection = styled(Box)`
+  border-style: solid;
+  border-right-style: none;
+  border-top-style: none;
+  border-width: 1px;
+  border-color: ${colors.border};
+  width: 100%;
+`;
+
+const ComponentToolboxItem = styled(Box)`
```

Comment:
Can you please use the Box props for this instead of a styled component?
If you need the gap prop - feel free to add it in the component library :)